test/integration_graph_test.go

```diff
@@ -86,38 +86,40 @@ func TestTerragruntApplyGraph(t *testing.T) {
 	t.Parallel()
 
 	testCases := []struct {
+		args               string
 		path               string
 		expectedModules    []string
 		notExpectedModules []string
 	}{
 		{
+			args:               "graph apply --terragrunt-non-interactive --terragrunt-working-dir %s --terragrunt-graph-root %s",
```

Comment:
I didn't see it, so I'm going to ask to be sure: Do we have a check to see if a user is using _both_ `--all` and `--graph`? What happens in that circumstance?

Reply:
This won't work. What to do in this case, return error `cannot use `--all` and `--graph` flags simultaneously`?

Reply:
Ya. Users have asked for the ability to do this, but we don't support it, so we should throw an error.

Reply:
Is this message OK?

```
Using the `--all` and `--graph` flags simultaneously is not supported.
```